src/utils/personOppgaveUtils.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  PersonOppgave,
+  PersonOppgaveType,
+} from "@/data/personoppgave/types/PersonOppgave";
+
+export const isBehandletOppgave = (personOppgave: PersonOppgave): boolean => {
+  return (
+    !!personOppgave.behandletTidspunkt && !!personOppgave.behandletVeilederIdent
+  );
+};
+
+export const hasUbehandletPersonOppgaveDialogmotesvar = (
+  personOppgaver: PersonOppgave[]
+): boolean => {
+  return (
+    personOppgaver.filter(
```

Comment:
Går an å bruke `.some` i stedet for `.filter` og sjekk på lengde 🙂

Reply:
Nice, skal bytte!